egs/librispeech/ASR/RESULTS-100hours.md

```diff
@@ -3,6 +3,15 @@
 This page shows the WERs for test-clean/test-other using only
 train-clean-100 subset as training data.
 
+## Distillation with hubert
+### 2022-05-27
+
+Decoding method is modified beam search. Epoch is 0-based when doing these experiments.
+|                                     | test-clean | test-other | comment                                  |
+|-------------------------------------|------------|------------|------------------------------------------|
+| baseline no vq distillation         | 7.08       | 18.66      | --epoch 19, --avg 10, --max-duration 200 |
+| distillation with hubert            | 5.68       | 15.80      | --epoch 19, --avg 10, --max-duration 200 |
```

Comment:
Please describe the training commands and decoding commands. And also please upload the training logs, decoding results, and pretrained model to huggingface and place a link here, so that others can reproduce your results.

Reply:
Will re-run both baseline and +vq experiments with stateless6. 
The results listed is from #339. This pr is a refactored of that.